clouds/bigquery/modules/sql/quadbin/QUADBIN_TOCHILDREN.sql

```diff
@@ -3,20 +3,43 @@
 ----------------------------
 
 CREATE OR REPLACE FUNCTION `@@BQ_DATASET@@.QUADBIN_TOCHILDREN`
-(quadbin INT64, resolution INT64)
+(cell INT64, children_resolution INT64)
 RETURNS ARRAY<INT64>
 AS ((
-    IF(resolution < 0 OR resolution > 26 OR resolution < ((quadbin >> 52) & 0x1F),
+    IF(children_resolution < 0 OR children_resolution > 26 OR children_resolution < ((children_resolution >> 52) & 0x1F),
         ERROR('Invalid resolution'),
         (
-            WITH __zxy AS (
-                SELECT `@@BQ_DATASET@@.QUADBIN_TOZXY`(quadbin) AS tile
+            WITH
```

Comment:
I think we can compact more the constants CTEs

Reply:
can you explain what do you mean?

Reply:
I mean instead of having 3 CTEs, have 2 or less